packages/shared/lib/integrations/scripts/connection/github-app-oauth-post-connection.ts

```diff
@@ -0,0 +1,17 @@
+import type { InternalNango as Nango } from './connection.manager.js';
+
```

Comment:
This post connection script for the OAuth is what allows us to know which user initiated the request and allows us to reconcile the user once the installation is approved and we receive the webhook